src/deploy/functions/release.js

```diff
@@ -117,7 +117,7 @@ module.exports = function(context, options, payload) {
   var projectId = context.projectId;
   var sourceUrl = context.uploadUrl;
   var appEngineLocation = getAppEngineLocation(context.firebaseConfig);
-  utils.logWarning("appEngineLocation " + appEngineLocation);
+  logger.debug(`appEngineLocation: ${appEngineLocation}`);
```

Comment:
I don't know why this was a `logWarning` as it seems like it shouldn't have been. Changed it to a debug, but it might be better to remove altogether? Was this just a stray log that wasn't caught at review time?

Reply:
as a user this is a pretty confusing message

Reply:
Pretty sure its a stray log, we should remove.